src/noisepy/seis/S1_fft_cc_MPI.py

```diff
@@ -70,7 +70,9 @@ def cross_correlate(
     # #########PROCESSING SECTION##########
     #######################################
 
-    tt0 = time.time()
+    tlog = TimeLogger(logger, logging.INFO)
+    # tlog.enabled = False
+    t_all = tlog.reset()
```

Comment:
Could we use a more descriptive variable here? `t_all` makes me think of setting a global time variable, not resetting the global log time.